main.go

```diff
@@ -79,11 +79,20 @@ func main() {
 		os.Exit(0)
 	}
 
-	s := createServer(*host, *port, *pacurl, a)
-	log.Printf("Listening on %s", s.Addr)
-	if err := s.ListenAndServe(); err != nil {
-		log.Fatal(err)
-	}
+	log.Fatal(gogroup(networks(*host), func(network string) error {
+		s := createServer(*host, *port, *pacurl, a)
+		addr := s.Addr
+		if addr == "" {
+			addr = ":http"
+		}
```

Comment:
I don't think line 86 is actually reachable, given that `*port` is an int (can't be an empty string) and we use net.JoinHostPort() (which will always insert a `:`). Do we need this check?

Reply:
Good point. I lifte the code directly from `net.ListenAndServer` without thinking too much about it. Fixed.